app/services/converters/order_detail_to_journal_rows.rb

```diff
@@ -0,0 +1,23 @@
+module Converters
+  class OrderDetailToJournalRows
+
+    attr_accessor :journal, :order_detail, :total
+
+    def initialize(journal, order_detail, options = {})
+      @journal = journal
+      @order_detail = order_detail
+      @total = options[:total] || order_detail.total
+    end
+
+    def to_a
+      [{
+        account: order_detail.product.account,
```

Comment:
I think it might be helpful to extract an overridable method that returns a hash of non-common fields. I think in the open case, it would just return `account`. Then you could merge that with description, order_detail_id, amount, and journal_id. That would probably make creating the converter class for NU easier, if we can just override that method.